ci.nix

```diff
@@ -3,14 +3,20 @@
 { supportedSystems ? [ "x86_64-linux" "x86_64-darwin" ]
 , ifdLevel ? 3
 # Whether or not we are evaluating in restricted mode. This is true in Hydra, but not in Hercules.
-, restrictEval ? false }:
-let
+, restrictEval ? false
+, checkMaterialization ? false }:
+ let
   inherit (import ./ci-lib.nix) dimension platformFilterGeneric filterAttrsOnlyRecursive;
   inherit (import ./default.nix { checkMaterialization = false; }) sources nixpkgsArgs;
```

Comment:
`nixpkgsArgs` from here is now shadowed, so maybe get rid of it. Then perhaps we should just pull sources from `nix/sources.nix` directly.